proposals/5/README.md

```diff
@@ -0,0 +1,103 @@
+****---
+title: Decomposition of Jenkins X
+linktitle: Decomposition of Jenkins X
+description: Decomposition of Jenkins X
+type: docs
+weight: 50
+---
+
+## 1. Overview
+
+This document outlines the work for decomposing Jenkins X into a series of reusable libraries.
+
+## 1.1 Motivation
+
+At the moment, the source code that comprises the main JX binary is tightly coupled and difficult to work on.  Changes have
+unintended consequences and this makes the codebase fragile.
+
+One of the design goals in Accelerate is to have a **Loosely Coupled Architecture** - in particular: 
```

Comment:
on a library level or on a service level? What I understand is that we are trying to split out libraries not services, so the coupling is not really loose.

Reply:
This is one part of many proposals that we're trying to create. I'm expecting another to be around "using actual microservices" or different binaries in controllers etc.  This will aid the loose coupling, but we need and API to communicate over first. 

Trying to avoid "boiling the ocean"